test/Driver/batch_mode_with_supplementary_filelist_unicode.swift

```diff
@@ -1,9 +1,25 @@
 // RUN: %empty-directory(%t)
 // RUN: echo 'print("Hello, World!")' >%t/main.swift
-// RUN: touch %t/𝔼-file-01.swift %t/😂-file-02.swift %t/Ω-file-03.swift
+// RUN: echo "" > %t/𝔼-file-01.swift 
```

Comment:
This avoid using gnu core utils touch which on windows doens't support utf16 arguments